File: Data Structures & Algorithms/duplicate-integer/submission-0.py
class Solution:
    def hasDuplicate(self, nums: List[int]) -> bool:
        b=0
        for i in range(len(nums)):
            a=nums[i]
            for j in range(i+1,len(nums)):
                if (a==nums[j]):
                    b+=1
        if b>0:
            return True
        else:
            return False

File: Data Structures & Algorithms/binary-search/submission-0.java
class Solution {
    public int search(int[] nums, int target) {
       int len=nums.length;
       int last=len-1;
       int first=0;
       
       while(first<=last)
       {
        int mid=(first+last)/2;
        
        if(target==nums[mid]){
            return mid;}

        if (target>nums[mid]){
            first=mid+1;
    }
        else
        {
        last=mid-1;
       }
}
 return -1;
}
}